src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -143,6 +143,7 @@ public static class Captions
       public static readonly string ObservedDataFolder = "Observed Data";
       public static readonly string ParameterIdentificationFolder = "Parameter Identifications";
       public static readonly string SensitivityAnalysisFolder = "Sensitivity Analyses";
+      public static readonly string QualificationPlanFolder = "Qualification Plans";
       public static readonly string LLOQ = "LLOQ";
```

Comment:
I think we will have only 1 Qualification plan per project. but still an open question so far, so let's keep it